tournamentmanager/progress_menu.py

```diff
@@ -339,11 +339,12 @@ def __init__(
         channel: discord.TextChannel,
         checkin_role: discord.Role,
         participant_role: discord.Role,
+        check_time,
     ):
         embed = discord.Embed(title="Check-in")
         embed.description = f"Le check-in est en cours dans le channel {channel.mention}"
         embed.add_field(name="Progression", value="Démarrage dans 10 secondes.", inline=False)
-        embed.add_field(name="Temps restant", value="0:30:00", inline=False)
+        embed.add_field(name="Temps restant", value="{0}".format(timedelta(seconds=check_time)), inline=False)
```

Comment:
J'ai du mal à comprendre l'intérêt 🤔 
```suggestion
        embed.add_field(name="Temps restant", value=str(timedelta(seconds=check_time)), inline=False)
```

Reply:
Ça c'est mon absence de connaissances des fonctions un peu avancés de python